### ISSUE TITLE
Should we fire unload handlers on portaled pages when the host page unloads?

### ISSUE THREAD

**Issue by domenic** [WICG/portals#239]
We have a [Chromium issue to do this](https://bugs.chromium.org/p/chromium/issues/detail?id=1038777). On the other hand, unload events are kind of bad, and not guaranteed in general. Maybe we should explicitly _not_ fire them?

**Comment by jeremyroman** [comment]
Yeah, this is kind of a mess. (As I've previously mentioned to @domenic it's a mess for iframes too, which makes it harder to decide what the real right behavior is.)
